mima/src/main/scala/org/typelevel/sbt/TypelevelMimaPlugin.scala

```diff
@@ -41,6 +41,11 @@ object TypelevelMimaPlugin extends AutoPlugin {
   )
 
   override def projectSettings = Seq[Setting[_]](
+    mimaReportBinaryIssues := {
+      if ((mimaReportBinaryIssues / skip).value)
+        ()
+      else mimaReportBinaryIssues.value
+    },
```

Comment:
This looks like a good upstream candidate.

Reply:
- https://github.com/lightbend/mima/issues/684